### PR TITLE
Documentation for layers

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for the issue #1417 

https://github.com/kartoza/sawps/assets/138113337/f5228704-1251-417d-af1d-0fbbfe38cadc

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 36 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 38 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 9 min 9 sec

File: docs/src/user/manual/explore/layers.md [hunk 1]
```diff
@@ -1,2 +1,127 @@
 # Layers
-<!-- To Be Populated -->
+
+Welcome to the documentation for map layers avalable in application. This document provides an overview of the map layers, how to access them, and how to interact with and customise their appearance.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/layers.md:3]:
```suggestion
Welcome to the documentation for map layers available in application. This document provides an overview of the map layers, how to access them, and how to interact with and customise their appearance.
```

File: docs/src/user/manual/explore/layers.md [hunk 2]
```diff
@@ -1,2 +1,127 @@
...
+
+![Layers](./img/layers-1.png)
+
+1. **Layers**: By clicking on the `LAYERS` user will able to see the available layers.
+
+2. **Map**: By clicking on the `MAP` user will able to see map visualisation.
+
+3. **Available Layers**: Available layers side panel, user can use this layers for better visualisation.
+
+# Layers with map
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/layers.md:15]:
```suggestion
## Map layers
```

File: docs/src/user/manual/explore/layers.md [hunk 3]
```diff
@@ -1,2 +1,127 @@
...
+
+    ![Displaying All layers on the Map](./img/layers-12.png)
+
+    1. **All Selected Layers**: All the selected layers on the side panel.
+
+    2. **Map Visualisation**: Map visualisation after selecting all layers.
+
+    3. **Names**: Popup displaying names of ecosystem type, critical biodiversity area, and protected area of pointed property.
+
+# Summary
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/layers.md:126]:
```suggestion
## Summary
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 25 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 42 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 37 sec

**Pull request merged**